cfme/tests/cli/test_appliance_console.py

```diff
@@ -975,7 +978,63 @@ def test_appliance_console_network_conf():
             9.
             10. Check IP and hostname values matches in Summary Information.
     """
-    pass
+    appliance = temp_appliance_preconfig_modscope
+    command_set = ("ap", RETURN)
+    old_result = appliance.appliance_console.run_commands(command_set, timeout=30, output=True)
+    logger.info('"ap" command output before test run:%s' % old_result)
+    first_screen_old = old_result[0].split("\n")
+    ipv4 = re.match(IPv4_REGEX, "".join([i for i in first_screen_old
```

Comment:
This is redundant - your regex string already includes the label text, why not match against the entire string?  No need to iterate over every line separately.

I would also use a named group, instead of just the magic `2` index.

Reply:
Thank you for the suggestion.